cypress/integration/catalog/categories.js

```diff
@@ -1,6 +1,8 @@
 /// <reference types="cypress"/>
 /// <reference types="../../support"/>
 
+import "cypress-wait-until";
```

Comment:
When I removed it - tests are working but when I leave this import then I cannot execute tests 
```suggestion

```

Reply:
I forgot to remove import after testing if this library will be helpful to solve problem with reload. This is additional library I've installed so failing in your case was expected.